tsconfig.json

```diff
@@ -1,16 +1,13 @@
 {
-	"compilerOptions": {
-		"target": "ES2015",
-		"module": "commonjs",
-		"declaration": true,
-		"outDir": "build",
-		"strict": true,
-		"removeComments": true
-	},
-	"include": [
-		"src/**/*"
-	],
-	"exclude": [
-		"src/**/*.test.ts"
-	]
-}
\ No newline at end of file
+  "compilerOptions": {
+    "target": "ES2015",
+    "module": "commonjs",
+    "declaration": true,
+    "outDir": "build",
+    "strict": true,
+    "removeComments": true,
+    "esModuleInterop": true
+  },
```

Comment:
apologies for the small, padantic nitpick, but could you change the indentation back to tabs or separate the formatting changes into its own commit? i know it doesn't make a difference to how the code runs, however, i want to avoid combining multiple unrelated changes in one commit if possible since it makes it easier to review if theres ever a need to go back later to see what changed